DP-8: Update Write the Docs (#5785)
OpenNMS/opennms: docs/modules/write-the-docs/pages/write-docs.adoc

Use the target's assigned ID to link to other parts of the documentation.
To refer to a target's assigned ID, use the following syntax:

[source, asciidoc]
<<doc-guidelines-links, Link name>>

If you need to link to another document and want to modify the link text, use the following syntax:

[source, asciidoc]
----
xref:link/to/doc.adoc[Document Name]

<<link/to/doc.adoc#tag-id, Section title>>
----

To include an external link, use the following syntax:

.Include external link with defined hyperlink text
[source, asciidoc]
http://www.opennnms.com[Link text here]

Links should have descriptive hyperlink text associated with them, and should avoid generic phrases (for example, "Click here").
This helps with search engine optimization.

> When inserting an external link to the OpenNMS documentation (for example, linking to the main docs from another project), you can link to the latest version of the docs by replacing the version number in the URL with `latest` (`\https://docs.opennms.com/horizon/latest/index.html`).

Author: To keep with our pattern of avoiding "doc":

```suggestion
When inserting an external link to the OpenNMS documentation (for example, linking to the main docs from another project), you should link to the latest version of the page.
To do so, replace the version number in the URL with `latest` (`\https://docs.opennms.com/horizon/latest/index.html`).
```
